syntax = "proto3";


service SimpleActions {
  rpc AddNumbers (SumRequest) returns (SumReply) {}
}


message SumRequest {
  int32 a = 1;
  int32 b = 2;
}


message SumReply {
  int32 c = 3;
}
